sdk/communication/communication-phone-numbers/test/public/utils/recordedClient.ts

```diff
@@ -6,14 +6,21 @@ import type { RecorderStartOptions, SanitizerOptions, TestInfo } from "@azure-to
 import { Recorder, env, isPlaybackMode } from "@azure-tools/test-recorder";
 import { PhoneNumbersClient } from "../../../src/index.js";
 import { parseConnectionString } from "@azure/communication-common";
-import type { TokenCredential } from "@azure/identity";
+import { type TokenCredential } from "@azure/identity";
 import { isNodeLike } from "@azure/core-util";
 import { createTestCredential } from "@azure-tools/test-credential";
 import { createMSUserAgentPolicy } from "./msUserAgentPolicy.js";
 import { createOperationLocationFixPolicy } from "./operationLocationFixPolicy.js";
+import { DefaultAzureCredential } from "@azure/identity";
+let fetch: typeof globalThis.fetch;
```

Comment:
`fetch` is available in NodeJS v20+ so we shouldn't need `node-fetch`.  Did you get errors when using it without `node-fetch`?